package/gluon-mesh-vpn-core/luasrc/usr/bin/gluon-wan

```diff
@@ -21,12 +21,13 @@ if ok ~= 0 then
 	io.stderr:write(string.format("gluon-wan: unable to change to group: %s\n", err))
 	os.exit(1)
 end
+local args = arg
+args[0] = args[1]
+table.remove(args, 1)
+print(args[0], unpack(args))
```

Comment:
This print can be removed, it was just for debug purposes.

Reply:
Thought the same, it is removed